src/gui/mrview/window.cpp

```diff
@@ -2012,7 +2013,7 @@ namespace MR
             return;
           }
 
-          assert ("shouldn't reach here!" && false);
+          assert (opt.opt->is ("info") or opt.opt->is ("debug") or ("shouldn't reach here!" && false));
```

Comment:
http://en.cppreference.com/w/cpp/keyword/or

Is [C++98](http://www.bezem.de/pdf/ReservedWordsInC.pdf) onward apparently.